source/adminguide/service_offerings.rst

```diff
@@ -379,19 +379,27 @@ To create a new disk offering:
       Storage for the volume to be provisioned. If no such primary
       storage exists, allocation from the disk offering will fail..
 
-   -  **Public**: Indicate whether the service offering should be available
-      all domains or only some domains. Choose Yes to make it available
-      to all domains. Choose No to limit the scope to a subdomain;
-      CloudStack will then prompt for the subdomain's name.
+   -  **Public**: Indicate whether the disk offering should be
+      available all domains or only some domains. Choose Yes to make it
+      available to all domains. Choose No to limit the scope to one or more
+      domains.  When 'Public' is set to 'no' a multi-selection list box is
+      displayed. One or more domains can be selected from this list box.
 
 #. Click Add.
 
+.. [2] These options are dependant on the capabilities of the hypervisor or the shared storage system which the VMs are on.
+   If the hypervisor or underlying storage don't support a particular capability in the offering, the setting will have no effect.
+
 
 Modifying or Deleting a Service Offering
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Service offerings cannot be changed once created. This applies to both
-compute offerings and disk offerings.
+Service offerings cannot be materially changed once created. This applies to 
+both compute offerings and disk offerings.  However their name, description 
+and scope can be modified. To edit the name or description navigate to the
+service offering's detail page and click on the edit icon |edit-icon.png|. 
+To alter the scope (zones and domains) that an offering is available in
```

Comment:
Since zone specific offerings are referred here, will it be better to mention zone field in create section as well?

Reply:
updated to include zones